app/uk/gov/hmrc/fhregistration/models/businessregistration/Identification.scala

```diff
@@ -17,10 +17,14 @@
 package uk.gov.hmrc.fhregistration.models.businessregistration
 
 import com.github.tototoshi.play.json.JsonNaming
-import play.api.libs.json.{Format, Json}
+import play.api.libs.json.JsonNaming.SnakeCase
+import play.api.libs.json.{Format, Json, JsonConfiguration}
 
 case class Identification(idNumber: String, issuingInstitution: String, issuingCountryCode: String)
 
 object Identification {
-  implicit val formats: Format[Identification] = JsonNaming snakecase Json.format[Identification]
+  implicit val formats: Format[Identification] = {
+    implicit val config: JsonConfiguration = JsonConfiguration(SnakeCase)
```

Comment:
Looks interesting, where did we get SnakeCase from? Presume it's in migration guide or somewhere similar